### DISCUSSION TITLE
Work in progress

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#687] [Announcements]
<img width="123" alt="layoutTest" src="https://user-images.githubusercontent.com/1558052/192142512-3bacfde7-f1ce-41c0-b930-f5a66836017f.png">

**Comment by euu2021** [comment]
Very useful!

**Comment by dpolivaev** [comment]
Status update
<img width="238" alt="layout-test" src="https://user-images.githubusercontent.com/1558052/196008646-17785891-9be3-4b32-966b-058c549351fd.png">

**Reply by mdchaves2** [reply]
Great! It looks really nice! I was waiting for this feature for more than 10 years! Could you release a preview version so that we can peform some tests?
